File: Hire-Quest(Backend)/java/com/alok91340/gethired/entities/Notification.java
/**
 * 
 */
package com.alok91340.gethired.entities;

import java.time.LocalDateTime;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

/**
 * @author aloksingh
 *
 */

@Entity
@Setter
@Getter
@NoArgsConstructor
public class Notification {
	
	@Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String title;
    private String body;
    private String senderUsername;
    private String receiverUsername;
    private String notificationType;
    private boolean readStatus;
    private LocalDateTime timestamp;
    
}

File: HireQuest(Frontend)/app/src/main/java/com/example/gethired/entities/Notification.kt
package com.example.gethired.entities

import java.time.LocalDateTime

data class Notification(
    val id: Long,
    val title: String,
    val body: String,
    val senderUsername: String,
    val receiverUsername: String,
    val notificationType: String,
    var readStatus: Boolean,
    val timestamp: String
)
